src/lib/auth/oauth-client.mjs

```diff
@@ -12,18 +12,6 @@ const clientSecret =
 const REDIRECT_URI = `http://127.0.0.1`;
 
 class OAuthClient {
```

Comment:
will jsdoc this in another pr, this one is getting big

Reply:
in the JSDoc PR can you add the annotation `//@ts-check` to the top of the file? it'll turn on in-IDE static analysis and should help you generate good types.